src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1667,6 +1693,49 @@ protected override void HandleModulesRequestAsync(IRequestResponder<ModulesArgum
             responder.SetResponse(response);
         }
 
+        // test -- need to delete
```

Comment:
>// test -- need to delete [](start = 8, length = 25)

Do you still want this comment here?

Reply:
This will be removed once all changes are completed.